src/Environment/MapCell.h

```diff
@@ -0,0 +1,23 @@
+#ifndef ALIFE_MAPCELL_H
+#define ALIFE_MAPCELL_H
+
+#include <vector>
+#include "src/Environment/Food.h"
+/*
+// Map Cells are the grid that cmake up the map
+// they provide locations for the organisims to move through
+// and contain the information that organisims need to make decisions
+*/
+class MapCell {
+    private:
+        int temperature;
+        int light;
+        std::vector<Food> food;
+    public:
+        MapCell(int inTemp, int inLight);
+        void mapCellStep();
+        int getTemperature();
```

Comment:
if we make all the factors an array `int factors[4]`, we can use one accessor function that takes in the index of the factor we want. Somewhere else make an enum with `TEMPERATURE, LIGHT, PLACEHOLDER0, PLACEHOLDER1`. We can figure out what the rest of the factors will be called later.